res/controllers/Pioneer-DDJ-SB2-scripts.js

```diff
@@ -10,6 +10,11 @@ PioneerDDJSB2.vinylModeOnStartup = false;
 // Allows toggling slip faster, but is counterintuitive.
 PioneerDDJSB2.invertVinylSlipButton = false;
 
+// If true, pressing shift + cue will play the track in reverse and enable slip mode,
+// which can be used like a censor effect. If false, pressing shift + cue jumps to
+// the beginning of the track and stops playback.
+PioneerDDJSB2.reverseRollOnShiftCue = false;
```

Comment:
Is this feature a temporary reverse roll, such that it resumes playing (slip mode) when the buttons are released?, or does it continue to play backwards until pressed again?